src/scripts/release_picard.sh

```diff
@@ -184,12 +184,12 @@ cd $TMPDIR
 # clone
 git clone $PICARDGITROOT picard
 cd picard
-ant clone-htsjdk
+ant -buildfile build-from-src.xml clone-htsjdk
```

Comment:
The idea of releasing only to OSS sounds good to me -- I imagine someone would still have to run a "git tag" command before each release, so the release workflow might just become: tag (and update version number in build.sbt?), release to OSS, and publish documentation via the script.

Is this simpler option do-able in this branch, or would you prefer to change the release process in a separate PR?